Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -529,14 +549,33 @@ class BocList
 
   private static FixValidationErrorOverflowSize(scrollableContainer: HTMLElement)
   {
-    // Sets the width of the validation error field to be as wide as the visible part of the bocList.
+    // Sets the width of the validation error field to be as wide as the visible part of the bocList
+    // minus how much space the td on the left requires to align with the error indicator column.
     // This enables sticky behaviour while still allowing for a scrollbar if required.
 
-    const width = LayoutUtility.GetWidth(scrollableContainer);
+    // -1 to account for rounding
+    const availableWidthInContainer = scrollableContainer.clientWidth - 1;
+
+    const td: Nullable<HTMLElement> = scrollableContainer.querySelector('div.bocListTableBlock tr.bocListValidationRow td:first-child');
+
+    let widthBeforeValidationFailureColumn = 0;
+    // width does not matter if the first td is already the one with the error messages (and therefore has child nodes)
+    if (td != null && !td.hasChildNodes())
+    {
+      widthBeforeValidationFailureColumn = Math.max(td.clientWidth + 1 - scrollableContainer.scrollLeft, 0);
+    }
 
     const overflowContainers = [...scrollableContainer.querySelectorAll<HTMLElement>(':scope tr.bocListValidationRow > td > div')];
-    overflowContainers.forEach(e => {
-      e.style.width = "calc(" + width.toString() + "px - var(--remotion-themed-scrollbar-width))";
+
+    overflowContainers.forEach(container => {
+
+      let style = window.getComputedStyle(container);
+
+      let margin = parseFloat(style.marginLeft) + parseFloat(style.marginRight);
+      let padding = parseFloat(style.paddingLeft) + parseFloat(style.paddingRight);
+      let border = parseFloat(style.borderLeftWidth) + parseFloat(style.borderRightWidth);
+
+      container.style.width = (availableWidthInContainer - widthBeforeValidationFailureColumn - margin - padding - border).toString() + "px";
```

Comment:
nitpick: this depends on using the correct `box-sizing` model. If you use `content-box`, the width is without the padding, border, and marging, and with `border-box`, the padding and border are part of the width. 

I'm not 100% sure I get the intention of the width calculation just from reading the code. what I see is that we get the margin+padding+border for the container element and then set a width minus all those. if we want to be 100% correct, we need to test which box-sizing is used and adjust accordingly. On the other hand, if we define that margin, padding and border of the root div must always be zero, we don't need to account for anything. 

I checked and it seems like container doesn't have any padding, border, or margin anyway. We could perhaps simply ignore this? or perhaps add an entry in CSS with a border, margin and padding on this element being `0 !important` plus a note. That way, it's not (easily) overrideable. The alternative is to also account for box-sizing changes here in the code (probably cleaenst?) or force those properties to zero in javascript (not preferred)

Reply:
This is supposed to fix the padding on the parent td, which is 1 by default, but I used the div instead of the td. Therefore I think we don't care about the box sizing as it should always add the padding to the inner width of a child element.